docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,55 @@ which groups these values were observed, as well as the difference and ratio
 between the maximum and the minimum values. For more information refer to the
 subpackage :code:`fairlearn.metrics`.
 
+Failure modes encountered when abstracting
+--------------------------------------------------
+
+In "Fairness and Abstraction in Sociotechnical Systems" Selbst et al. [#4]_ identify failure modes that arise the applying abstraction. They identify them as:
+
+* *The Framing Trap*
+
+* *The Portability Trap*
+
+* *The Formalism Trap*
+
+* *The Ripple Effect Trap*
+
+* *The Solutionism Trap*
+
+These traps are inspired by the observation that each of these traps are the result of failing to consider the way in with social context and technology are interrelated, as well as a deeper understanding of "the social" in order to solve problems Ackerman [#5]_. 
+
+The Framing Trap
+^^^^^^^^^^^^^^^^^^^^
+
+Selbst et al. [#5]_ define this as a "failure to model the entire system over which a social criterion, such as fairness, will be enforced".
+
+Examples include:
+
+Labelling in a risk assessment setting. Risk assessment may be used as a predictive model whose goal is to assess the risk of an individual to ease the decision-making process. For example, using risk assessment to decide whether a person is eligible for morgtage approval. Another example is using risk assessment to decide whether a person is likely to re-engage in criminal behaviour as they move through the criminal justice pipeline. 
```

Comment:
```suggestion
Labelling in a risk assessment setting. Risk assessment may be used as a predictive model whose goal is to assess the risk of an individual to ease the decision-making process. For example, using risk assessment to decide whether a person is eligible for mortgage approval. Another example is using risk assessment to decide whether a person is likely to re-engage in criminal behavior as they move through the criminal justice pipeline. 
```

Reply:
Only noticed this after fixing typos. They appear to be fixed. let me know if they still pop up.